Code/sym.py

```diff
@@ -0,0 +1,34 @@
+# SE HW - 2
+
+class Sym:
+    def __init__(self, column_name, column_position):
+        self.column_name = column_name
+        self.column_position = column_position
```

Comment:
Give default values as c or 0 and s or '', just to keep the format consistent with lua. You can refer to my PR: https://github.com/Surya-06/SE-Proj22-Team11/pull/21/files

Reply:
As discussed, this will be taken up on a different thread.